elk/run.py

```diff
@@ -44,6 +45,8 @@ def __post_init__(self):
             # Save in a memorably-named directory inside of
             # ELK_REPORTER_DIR/<model_name>/<dataset_name>
             ds_name = ", ".join(self.cfg.data.prompts.datasets)
+            if self.custom_ds_n:
+                ds_name = self.custom_ds_n
             root = elk_reporter_dir() / self.cfg.data.model / ds_name
```

Comment:
I think we would need to do the same for models too.
However, I feel like, what we want instead maybe, is to be able to set a custom path for the whole root ... or at least what follows after elk_reporter_dirt()